### PR TITLE
Remove the ctr command

### PR DESCRIPTION
**Pull request by soharaki**
#7

### CODE REVIEW

File: build.go [hunk 1]
```diff
@@ -61,30 +60,46 @@ func buildAction(clicontext *cli.Context) error {
...
+
+	// Load images from buildkit tar stream
+	client, ctx, cancel, err := newClient(clicontext)
+	if err != nil {
 		return err
 	}
-	return ctrCmd.Wait()
+	defer cancel()
+
+	// FIXME: I wonder selecting the image digest type.
```
Comment (AkihiroSuda) [inline build.go:80]: What do you mean and who am "I"?

File: build.go [hunk 2]
```diff
@@ -61,30 +60,46 @@ func buildAction(clicontext *cli.Context) error {
...
+	var opts []containerd.ImportOpt
+	opts = append(opts, containerd.WithDigestRef(archive.DigestTranslator(sn)))
+
+	imgs, err := client.Import(ctx, buildctlStdout, opts...)
+	if err != nil {
+		return err
+	}
+
+	// Wait exporting images to containerd
+	buildctlCmd.Wait()
```
Comment (AkihiroSuda) [inline build.go:90]: Please make sure to check err value from `buildctlCmd.Wait`

File: build.go (hunk 1 above)
Comment (soharaki) [reply build.go:80]: I've forgotten.

Comment (soharaki) [reply build.go]: I'm terribly sorry for the unremoved comment.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**